src/contracts/core/DelegationManager.sol

```diff
@@ -305,36 +305,36 @@ contract DelegationManager is
         });
     }
 
-    /// @inheritdoc IDelegationManager
-    function decreaseBeaconChainScalingFactor(
+    function decreaseDelegatedShares(
         address staker,
-        uint256 existingDepositShares,
-        uint64 proportionOfOldBalance
+        uint256 curDepositShares,
```

Comment:
nit: `increaseDelegatedShares` calls this `existingDepositShares`. I have a slight preference for `cur`, but we should make consistent

Reply:
ty, good callout. i meant to change all instances for consistency as i also prefer `cur`

Reply:
[4d5d8a8](https://github.com/Layr-Labs/eigenlayer-contracts/pull/892/commits/4d5d8a86da20c26e4ae7c3e500b24e0438a880c6)